pages/osx/codesign.md

```diff
@@ -0,0 +1,11 @@
+# codesign
+
+> Create and manipulate code signatures for macOS.
+
+- Sign for an application with your certificate:
+
+`codesign -s {{"My Company Name"}} {{/path/to/App.app}}`
+
+- Verify a certificate to an application:
```

Comment:
The wording can be improved here also. If this means the the application's cert will be verified, then I believe "Verify the certificate of an application" explains the concept better.